ujin/programmers/day03/K번째수.js

```diff
@@ -0,0 +1,11 @@
+function solution(array, commands) {
+  let answer = [];
```

Comment:
배열 매서드를 사용하여 배열의 요소를 추가, 삭제를 하는 경우 `const` 로 선언해도 무방합니다.
```suggestion
  const answer = [];
```